packages/mui-material/src/DialogTitle/DialogTitle.d.ts

```diff
@@ -10,6 +10,7 @@ export interface DialogTitleTypeMap<
   D extends React.ElementType = TypographyTypeMap['defaultComponent'],
 > {
   props: P & {
+    ariaLabelledby?: string;
```

Comment:
There is no need to introduce a new `ariaLabelledBy` prop.

Reply:
Do we really need that `idProp` ? Because I don't see such thing anywhere else.

Reply:
I am not sure I understand. `idProp` is the ID if the dialog title component.